js/main.js

```diff
@@ -3,31 +3,72 @@ var summonBtn = document.querySelector('.summon-btn');
 
 var summonImg = document.querySelector('.image');
 
-var acceptBtn = document.querySelector('.accept-btn');
+var acceptBtnDiv = document.querySelector('.accept-btn-div');
+
+var rejectBtn = document.querySelector('.reject-btn');
 
+var acceptBtn = document.querySelector('.accept-btn');
 
-function getLoadFox(){
+var image = '';
 
-  var image = ''
+function getLoadFox() {
 
   var xhr = new XMLHttpRequest();
-  xhr.open('GET', 'https://randomfox.ca/floof/');  // http://shibe.online/api/shibes?count=[1-100]&urls=[true/false]&httpsUrls=[true/false]
+  xhr.open('GET', 'https://randomfox.ca/floof/');
 
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
 
-    console.log(xhr.status);
-    console.log(xhr.response);
+    // console.log(xhr.status);
+    // console.log(xhr.response);
```

Comment:
Please remove commented code prior to submitting for review.

Reply:
I was going to code that up in the next feature since the user can only add foxes so far, but not release them. 
I can do it for this one though